.travis.yml

```diff
@@ -1,27 +1,40 @@
 language: ruby
-sudo: false
+os: linux
 dist: bionic
 
-services:
-  - postgresql
-
 cache:
   bundler: true
 
 addons:
-  postgresql: "9.6"
+  postgresql: "11"
+  apt:
+    update: true
+    packages:
+      - postgresql-11
+      - postgresql-client-11
+      - postgresql-server-dev-11
 
 before_install:
-  - gem update --system 3.1.2
-  - gem install bundler
+  - sudo cp /etc/postgresql/{9.6,11}/main/pg_hba.conf
+  - sudo service postgresql restart 11
+  - gem update --system 3.1.4
+  - gem install bundler:2.1.4
+
+before_script:
```

Comment:
Do we want to try and move any of this back into Rakefile?

Reply:
Seems like trying to set this up via the rake file was not working correctly with travis. Not sure why but it fails more when add these tasks in the rake file vs setting it up in the travis.yml file. I think its more of a rails issue but it could also be a travis issue